### PR TITLE
helm-chart: Align README with actual default values

### PR DESCRIPTION
**Pull request by JamesGuthrie**

### CODE REVIEW

File: helm-chart/README.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ Following are the default configuration values:
 ```yaml
 connection:
   user: postgres
-  password: ""
-  host: db.timescale.svc.cluster.local
+  password: secretPassword
```
Comment (paulfantom) [inline helm-chart/README.md:40]: This should be defined as `""` in values.yaml instead.

File: helm-chart/README.md [hunk 2]
```diff
@@ -37,8 +37,8 @@ Following are the default configuration values:
...
+  host: timescaledb.monitoring.svc.cluster.local
```
Comment (paulfantom) [inline helm-chart/README.md:41]: Let's change this to `timescaledb.default.svc.cluster.local` here and in `values.yaml`. This way we won't be advocating to put timescaledb into `monitoring` namespace.

**Review by paulfantom — CHANGES_REQUESTED** [review]

File: helm-chart/README.md (hunk 1 above)
Comment (JamesGuthrie) [reply helm-chart/README.md:40]: done

File: helm-chart/README.md (hunk 2 above)
Comment (JamesGuthrie) [reply helm-chart/README.md:41]: done

**Review by paulfantom — APPROVED** [review]
lgtm

**Review by cevian — APPROVED** [review]

**Pull request merged**